frontend/src/metabase/query_builder/selectors.js

```diff
@@ -551,10 +551,9 @@ const isZoomingRow = createSelector(
   index => index != null,
 );
 
-export const getMode = createSelector(
-  [getLastRunQuestion],
-  question => question && getQuestionMode(question),
-);
+export const getMode = createSelector([getLastRunQuestion], question => {
```

Comment:
will revert with other feedback... i originally changed then and thought i undid the changes